tools/tools.go

```diff
@@ -0,0 +1,10 @@
+// +build tools
+
+// This package contains import references to packages required only for the
+// build process.
```

Comment:
Is the purpose of these references ensuring that modules recognize that these packages are used?

Reply:
Added a link to https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module which explains the technique and motivation.